rfml.go

```diff
@@ -476,6 +464,120 @@ func uploadRFMLFilesInDirectory(rfmlDirectory string) error {
 	return nil
 }
 
+func downloadRFML(c cliContext) error {
+	testDirectory := c.String("test-folder")
```

Comment:
any specific reason to use `test-folder` here @epaulet  ... was this part of the new spec?